Bottom: 439398e24f402d62b9d3258fd8d18f105377d6e0
Top:    231b16ab6a7850efff81eb9ac4f48dcf7aeb7a22
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-14 01:27:37 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
